src/driver/main.cpp

```diff
@@ -65,6 +66,12 @@ struct loader
            ap.help("Dim of a parameter (format: \"@name d1 d2 dn\")"),
            ap.append(),
            ap.nargs(2));
+
+        ap(output_names,
+           {"--output-names"},
+           ap.help("Names of node output (format: \"name_1 name_2 name_n\")"),
+           ap.append(),
+           ap.nargs(2));
```

Comment:
So what happens when the user doesn't pass in the output_names?

Reply:
Currently it will just default to the last instruction in the program (same as before this PR)